dapp-viem/README.md

```diff
@@ -0,0 +1,41 @@
+# DApp built with Viem connected to Hedera Testnet
+
+This code snippet demonstrates how to build a minimal DApp using Viem
+that connects to Hedera Testnet.
+
+## Running the DApp
+
+Ensure that you have Metamask installed, with network config,
```

Comment:
In addition to MM installed,  just mention having the Hedera portal account (with ECDSA keys) imported in MM. 
Network switching and config is better done in code.

Reply:
yep, I'll address this together with https://github.com/hedera-dev/hedera-code-snippets/pull/14/files#r1476021137 (in a follow up PR)